deploy/roles/kerberos/tasks/main.yml

```diff
@@ -51,3 +51,4 @@
     recursive: yes
     state: present
   with_items: "{{ log_file_locations }}"
```

Comment:
@stirlingv I think we might want to "namespace" these variables in general, so we don't run into the same issue we saw with elasticsearch.  So this one could be `kerberos_log_file_locations`.